src/DependencyGraph/RangeVertex.ts

```diff
@@ -70,10 +70,8 @@ export class RangeVertex {
    * @param cacheKey - key to retrieve from the cache
    * @param criterionString - criterion text (ex. '<=5')
    */
-  public getCriterionFunctionValue(cacheKey: string, criterionString: string): any | null {
-    const values = this.getCriterionFunctionValues(cacheKey)
-    const value = values.get(criterionString)
-    return value ? value[0] : null
+  public getCriterionFunctionValue(cacheKey: string, criterionString: string) {
```

Comment:
Type disapeard
```suggestion
  public getCriterionFunctionValue(cacheKey: string, criterionString: string): ?? {
```

Reply:
the type was `any | null` so basically `any`, which is not really informative (and triggers linter...)

Reply:
I see. Better without type then